packages/http-client-java/README.md

```diff
@@ -1,6 +1,6 @@
-# TypeSpec Java Emitter Library
+# @typespec/http-client-java
 
-This is a TypeSpec library that will emit a Java SDK from TypeSpec.
+TypeSpec library for emitting Java client from the TypeSpec REST protocol binding
 
 ## Prerequisite
```

Comment:
I've put back the `## Prerequisite` section.

Let me know whether there is some way to "customize" part of the generated README.